### ISSUE TITLE
[Request] Number of Months for a Resub for Manual Resub

### ISSUE THREAD

**Issue by BarryCarlyon** [justintv/Twitch-API#341]
Hello!

For resubs that are manual, this often results in a TwitchNotify of x has subbed for y months.

But in the API end point you would see the subscription as being created today!
Please can the number of months be added to the API end point?

**Comment by Myrathi** [comment]
Agreed! This would be exceptionally useful information via this endpoint.

**Comment by night** [comment]
I am doubtful you'll ever see this kind of data, at least not on the current API endpoint. There is likely an internal mechanism that counts subscriptions (including expired subscriptions) to the channel when a subscription renews. This API endpoint only provides you with active subscriptions to a channel.
